### ISSUE TITLE
I can't download  fish data for North America separately

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#5013]
**I can&#39;t download fish data for North America separately**

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/e4d5c249cc9b5b02e9751fdcdb86f6a6:ee8a3ce49a193ef50e971d9b57c121d67aa611eb24dd646cd5bcea9a17cb780153ab913368068bfd14a0ff5237ccef82827c5c028e04c60cd7ac78f1940027e2fa98a20750507eedaebc0d5078c6024d5daeaa17d84f250f64a0f060441434a9) - [Send email](https://www.gbif.org/api/feedback/user/mailto/e4d5c249cc9b5b02e9751fdcdb86f6a6:ee8a3ce49a193ef50e971d9b57c121d67aa611eb24dd646cd5bcea9a17cb780153ab913368068bfd14a0ff5237ccef82827c5c028e04c60cd7ac78f1940027e2fa98a20750507eedaebc0d5078c6024d5daeaa17d84f250f64a0f060441434a9)
System: Chrome 118.0.0 / Windows 10.0.0
Referer: https://www.gbif.org/occurrence/download/0023050-231002084531237
Window size: width 2512 - height 1310
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-10-22T07:29:56.119Z',to:'2023-10-22T07:35:56.119Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-10-22T07:29:56.119Z',to:'2023-10-22T07:35:56.119Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL

**Comment by MortenHofft** [comment]
@muttcg  I'm not sure if I've labelled this correctly, but it looks like a download issue

**Comment by ManonGros** [comment]
Note that the user is querying:
```
{
      "type": "equals",
      "key": "CONTINENT",
      "value": "North America",
      "matchCase": false
    }
```

shouldn't it be `NORTH_AMERICA`?

**Comment by MattBlissett** [comment]
Yes. I wonder if we should parse values of enumerations/vocabularies and return a 400 error if they don't make sense.
